package.json

```diff
@@ -268,12 +271,14 @@
 		"jsdom": "22.1.0"
 	},
 	"scripts": {
-		"build": "npm run build:packages && wp-scripts build",
+		"build": "npm run build:packages && concurrently 'npm run build:plugins' 'wp-scripts build'",
 		"build:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
 		"build:package-types": "node ./bin/packages/validate-typescript-version.js && ( tsc --build || ( echo 'tsc failed. Try cleaning up first: `npm run clean:package-types`'; exit 1 ) ) && node ./bin/packages/check-build-type-declaration-files.js",
 		"prebuild:packages": "npm run clean:packages && lerna run build",
 		"build:packages": "npm run --silent build:package-types && node ./bin/packages/build.js",
-		"build:plugin-zip": "bash ./bin/build-plugin-zip.sh",
+		"build:plugins": "npm run build --workspace ./plugins",
+		"build:plugin-zip": "concurrently 'npm run build:plugin-zip:plugins' 'bash ./bin/build-plugin-zip.sh'",
+		"build:plugin-zip:plugins": "npm run plugin-zip --workspace ./plugins",
```

Comment:
How does this work, does it build multiple zips for plugins, how to build a single plugin zip?

Reply:
It loops through the plugins directory and runs the `plugin-zip` script defined in each package json.

The Time to Read Block is using the vanilla `"plugin-zip": "wp-scripts plugin-zip",`, which is what I'd anticipate most plugins doing. The end result is a separate zip for each plugin inside the plugin's directory (e.g. `plugins/time-to-read-block/time-to-read-block.zip`).